src/app/errors/routes.ts

```diff
@@ -7,26 +7,23 @@ type Params = {
   url: string;
 };
 
+const loadableArgs = {
+  loader: () => import(/* webpackChunkName: "LoaderCentered" */ './components/LoaderCentered'),
+  loading: () => null,
+  modules: ['LoaderCentered'],
+  webpack: /* istanbul ignore next */ () => [(require as any).resolveWeak('./components/LoaderCentered')],
+};
+
 export const notFound: Route<Params> = {
-  component: Loadable({
```

Comment:
Are these changes for the navigation specs?

Reply:
This was a coverage issue. Some changes got into main where coverage was incomplete. This gets back to complete. I just extracted the (duplicated) args so that the `loading` method only had to get exercised once.